plugins/notifications-backend/src/service/types.ts

```diff
@@ -8,6 +8,8 @@ export type CreateNotificationRequest = {
   targetGroups?: string[];
 };
 
+// Keep in sync with FE: plugins/notifications-frontend/src/api/notificationsApi.ts
+// TODO: Once we have more to share, we will introduce a notifications-common library.
```

Comment:
let's keep the TODO in the doc. let the code document what exists rather than what will be in the future

Reply:
Having it copy&pasted is a workaround, it's easy to skip this change when we modify the type here. The document will not help us much.

I have created FLPATH-560 jira issue to introduce the notifications-common but recently let's wait with it, it would be overkilling solution for sharing just two types so far.

Reply:
I meant just the comment with the TODO. the comment that tells the developer to sync with another file is definitely needed.